### PR TITLE
feat: [IXSPD1-1697] started page for user role

### PR DESCRIPTION
**Pull request by galvin96**
## Description

Please describe the purpose of this pull request.

## Changes

- started payout event page for user role

## Attached Links

1. Jira ticket: [IXSPD1-1697]
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

[IXSPD1-1697]: https://investax.atlassian.net/browse/IXSPD1-1697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/PayoutItem/MoreEventBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { MEDIA_WIDTHS } from 'theme'
+
+const MorePayoutEvents = ({ payoutId }: { payoutId: number }) => {
+  const [payouts, setPayouts] = useState([] as PayoutEvent[])
+  const [page, setPage] = useState(1)
+  const [totalPages, setTotalPages] = useState(0)
+
+  useEffect(() => {
+    async function load() {
+      const data = await getPayouts({ page, offset: 3 })
```
Comment (thi-investax) [inline src/pages/PayoutItem/MoreEventBlock.tsx:18]: why we hardcode `3` here @galvin96 ?

Comment (thi-investax) [reply src/pages/PayoutItem/MoreEventBlock.tsx:18]: I think we should handle API errors properly otherwise this might break

Comment (galvin96) [reply src/pages/PayoutItem/MoreEventBlock.tsx:18]: This is my concern as well. The pagination module uses the offset as the page size. In this case, we would like to show 3 items per page, so I changed to 3 of offset. Docs: https://github.com/yonycalsin/nestjs-sequelize-paginate

Comment (thi-investax) [reply src/pages/PayoutItem/MoreEventBlock.tsx:18]: what about api error handling?

**Pull request merged**